Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -124,6 +124,30 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(output, expected)
     }
 
+    func testGlobalMaxPool1D() {
+        let layer = GlobalMaxPool1D<Float>()
```

Comment:
Could you make sure tests compile locally? This does not compile because the type was defined as `GlobalMaxPooling1D`, not `GlobalMaxPool1D`.